Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -53,6 +53,7 @@ $azCopyLocation = [System.IO.Path]::GetDirectoryName($azCopyExeLocation)
 # Initialize Azure.
 Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
 Initialize-Azure
+Import-Module $PSScriptRoot\ps_modules\VstsTaskSdk\VstsTaskSdk.psm1 -NOClobber
```

Comment:
You can directly specify path till folder while importing the module. No need to specify .psm1 file.

Reply:
Since AFC task is written with PS3 handler. It should already have VstsTaskSdk